gallery/workbook/microsoft.experimentation-experimentworkspaces.json

```diff
@@ -15,10 +15,10 @@
 				},
 				{
 					"id": "Workbooks/Online Experimentation/Experiment Analysis",
-					"name": "Experiment Analysis (preview)",
+					"name": "Experiment Analysis",
```

Comment:
Should we keep "preview" in the name? This workbook will be part of private preview and then public preview, before I'd expect this to be dropped.